### ISSUE TITLE
Documenting test process as a step

### ISSUE THREAD

**Issue by karlgroves** [w3c/wai-wcag-em#174]
My only real criticism of WCAG-EM is that WCAG-EM still drops the same ball that WCAG 2.x does by not having more granular details on the test criteria.   WCAG is structured as Principle -> Guideline -> Success Criteria, with no normative definition of *how* one may meet the criteria.     As a result, not only are the mechanism of meeting the SC not defined, nor are the methods of judging conformance.

Clearly, it is beyond the scope of WCAG-EM to take that on.  However, the document should include steps and descriptions for the evaluator(s) to come up with their own itemized criteria against which the product is judged.

IMO this is a critical missing piece of WCAG-EM

**Comment by stevefaulkner** [comment]
> the document should include steps and descriptions for the evaluator(s) to come up with their own itemized criteria against which the product is judged.

@karlgroves thanks do you have some words in mind?

**Comment by hidde** [comment]
I understand this is where evaluators struggle, but like you say @karlgroves, it's beyond the scope of WCAG-EM to take on the mechanism of meeting the SCs (or conformance), that's what Techniques, Understanding docs and WCAG's conformance section try to do. 

Do you think we can write up the itemised criteria you suggest globally, so that they work for all of WCAG?

**Comment by karlgroves** [comment]
I think what I'm getting at is largely contained in the structure of the [ACT Rules](https://www.w3.org/WAI/standards-guidelines/act/rules/)

Fundamentally, what evaluators need to know are:

* Does this check item apply to what I'm testing?
* What am I testing for?
* How do I test for it?
* If it fails, to which WCAG SC does it apply?

All of this is covered by ACT Rules. Unfortunately, the current ACT rules don't cover all SCs, nor do they provide complete coverage for more than a few SCs. That said, maybe this note could include language stating that the evaluator should base their evaluation criteria on a formally defined, well-structured set of checks that include applicability, expected outcome, and evaluation steps.

**Comment by hidde** [comment]
> All of this is covered by ACT Rules. 

yup

> Unfortunately, the current ACT rules don't cover all SCs, nor do they provide complete coverage for more than a few SCs.

Yes. Agree it's unfortunate, do think ACT rules would be the best place for it. I understand they have big workload and hard problems. I will continue to cheer on the initiative. 

> That said, maybe this note could include language stating that the evaluator should base their evaluation criteria on a formally defined, well-structured set of checks that include applicability, expected outcome, and evaluation steps.

I like that!
